sgid_client/SgidClient.py

```diff
@@ -8,8 +8,8 @@
 from .error import Errors, get_network_error_message, get_www_authenticate_error_message
 from .util import (
     convert_to_pkcs8,
-    is_stringified_array_or_object,
-    safe_json_parse
+    isSgidUserinfoObject,
```

Comment:
nit: do you want to standardise python naming convention?
aka.
`is_sgid_userinfo_object`
`parse_individual_data_value`

Reply:
fixed! I'm using a new tool (graphite.dev) which encourages less commits per PR so i've rebased the original commit [396ad25](https://github.com/opengovsg/sgid-client-python/pull/42/commits/396ad257e153840628f4ebbbc22b7676f1c37cfd)